src/oc/web/components/secure_activity.cljs

```diff
@@ -21,15 +21,15 @@
 (def default-activity-header-height 69)
 (def default-activity-content-height 136)
 
-(rum/defcs secure-activity < ;; Mixins
-                             rum/reactive
-                             (ui-mixins/render-on-resize save-win-height)
+(rum/defcs secure-activity < rum/reactive
                              ;; Derivatives
                              (drv/drv :secure-activity-data)
                              (drv/drv :made-with-carrot-modal)
                              ;; Locals
                              (rum/local 0 ::win-height)
                              (rum/local nil ::win-resize-listener)
+                             ;; Mixins
```

Comment:
Does this not work without the correct `rum/local` variables?  Just wondering why changing the order fixed this issue.